rectools/models/implicit_knn.py

```diff
@@ -97,16 +97,16 @@ def _recommend_for_user(
         if sorted_item_ids is not None:
             sorted_filtered_item_ids = sorted_item_ids[~fast_isin_for_sorted_test_elements(sorted_item_ids, viewed_ids)]
             n_items = user_items.shape[1]
-            reco_scores = self.model.recommend(user_id, user_items, N=n_items, filter_already_liked_items=False)
-            reco = np.array([r[0] for r in reco_scores])
-            scores = np.array([r[1] for r in reco_scores])
+            reco, scores = self.model.recommend(
```

Comment:
Isn't it possible to pass a batch of users to `recommend` method now?
Possible it would be faster

Reply:
Let's keep it to next PR?

Reply:
Ok